lib/linguist/languages.yml

```diff
@@ -3144,6 +3144,14 @@ Kusto:
   tm_scope: source.kusto
   ace_mode: text
   language_id: 225697190
+Kvlang:
```

Comment:
The proper name seems to be [KV or Kv](https://kivy.org/doc/stable/guide/lang.html) (inconsistent, so probably better to chose the all caps one). the samples folder would need to likewise change

Reply:
@tshirtman @matham what do u say ? KV or Kvlang, the reason i choose Kvlang was because the extension is also `.kv`

Reply:
@Nixinova its `Kvlang`, the docs will be updated to say the same soon

Reply:
We usually refer to the language as "kvlang", at least when we try to be technically precise, but yes, the documentation is inconsistent on that.

Reply:
The mixed case Kv looks the most odd to me. What @tshirtman wrote kvlang feels natural. KVlang feels slightly less natural but still better than Kvlang. At least to my eyes.